packages/icons/icons.yml

```diff
@@ -8549,7 +8888,9 @@
     - 32
 - name: monster
   friendly_name: monster
```

Comment:
All friendly names seem to be capitalized - I'm assuming this on too? 

  `friendly_name: Monster`

Reply:
Yes. Please initial cap all friendly names